wormhole-connect/src/views/Bridge/TransferLimitedWarning.tsx

```diff
@@ -1,25 +1,219 @@
 
 const TransferLimitedWarning = () => {
+  const { classes } = useStyles();
+  const [open, setOpen] = React.useState(true);
   const isTransferLimited = useIsTransferLimited();
+
+  const onClose = () => {
+    setOpen(false);
+  };
+
+  const onContinue = () => {
+    setOpen(false);
+    setTimeout(() => {
+      window.scrollTo(0, document.body.scrollHeight);
+    }, 500);
+  };
+
   if (
     isTransferLimited.isLimited &&
     isTransferLimited.reason &&
     isTransferLimited.limits
   ) {
     const chainName = wh.toChainName(isTransferLimited.limits.chainId);
-    const message =
-      isTransferLimited.reason === 'EXCEEDS_MAX_NOTIONAL'
-        ? `This transfer's estimated notional value would exceed the notional value limit for transfers on ${chainName} (${isTransferLimited.limits.chainNotionalLimit}) and may be subject to a 24 hour delay.`
-        : isTransferLimited.reason === 'EXCEEDS_LARGE_TRANSFER_LIMIT'
-        ? `This transfer's estimated notional value may exceed the notional value for large transfers on ${chainName} (${isTransferLimited.limits.chainBigTransactionSize}) and may be subject to a 24 hour delay.`
-        : isTransferLimited.reason === 'EXCEEDS_REMAINING_NOTIONAL'
-        ? `This transfer's estimated notional value may exceed the remaining notional value available for transfers on ${chainName} (${isTransferLimited.limits.chainRemainingAvailableNotional}) and may be subject to a delay.`
-        : '';
-    return <AlertBanner show={!!message} content={message} error />;
+
+    let message, title;
+    if (
+      isTransferLimited.reason === 'EXCEEDS_MAX_NOTIONAL' ||
+      isTransferLimited.reason === 'EXCEEDS_REMAINING_NOTIONAL'
+    ) {
+      // TODO: See if its necessary a different message for EXCEEDS_REMAINING_NOTIONAL case
+      title = (
+        <>
+          This transaction will take up to{' '}
+          <span className={classes.orange}>24 hours</span> to complete
+        </>
+      );
+      message = (
+        <>
+          This transaction will take up to 24 hours to complete. This{' '}
+          transaction will take up to 24 hours to process as Wormhole has{' '}
+          reached the daily limit for {chainName}.
+          <br />
+          <br />
+          This is a normal and temporary security feature by the Wormhole{' '}
+          network.
+          <br />
+          <br />
+          <a href={learnMoreLink} target="_blank" rel="noreferrer">
+            Learn more
+          </a>{' '}
+          about this temporary security measure.
+        </>
+      );
+    } else if (isTransferLimited.reason === 'EXCEEDS_LARGE_TRANSFER_LIMIT') {
+      title = (
+        <>
+          This transaction requires{' '}
+          <span className={classes.orange}>24 hours</span> to complete
+        </>
+      );
+      message = (
+        <>
+          This transaction requires 24 hours to complete. This transaction will{' '}
```

Comment:
Can we remove the first sentence? It's mentioned that it will take 24 hours in the second sentence already

Reply:
Done!